cmd/limactl/gendoc.go

```diff
@@ -101,7 +101,26 @@ and $LIMA_WORKDIR.
 	return doc.GenManTree(cmd.Root(), header, dir)
 }
 
+func escapeMarkdown(text string) string {
+	lines := []string{}
+	for _, line := range strings.Split(text, "\n") {
+		line = strings.ReplaceAll(line, "#", "\\#")
```

Comment:
Why only escape `#`? Shouldn't we escape all characters that might be interpreted by Markdown?

```
\ ` * _ [ ] ( ) # + - . |
```

Reply:
Escaping periods and dashes seems a bit paranoid, at least for the text that we currently have...

Reply:
The output is only used to generate HTML, so does it matter if we are over-escaping?

Reply:
Sure, it just felt a bit much with the escape at the end of each sentence\\.